gpu-operator/google-gke.rst

```diff
@@ -160,4 +275,6 @@ Related Information
 
 * If you have an existing GKE cluster, refer to
   `Add and manage node pools <https://cloud.google.com/kubernetes-engine/docs/how-to/node-pools>`_
-  in the Google Kubernetes Engine documentation.
\ No newline at end of file
+  in the GKE documentation.
+* When you create new node pools, specify the ``--node-labels="gke-no-default-nvidia-gpu-device-plugin=true"`` CLI argument
```

Comment:
... and let's also mention to disable the automatic GPU driver installation, i.e.
```
--accelerator type=...,gpu-driver-version=disabled``
```